examples/platform/silabs/rgb_led/RGBLEDWidget.cpp

```diff
@@ -0,0 +1,353 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
```

Comment:
NIT: copy & paste, but probably not google and then we can update year as well.
or is it copied from somewhere?